sdks/go/test/integration/primitives/state.go

```diff
@@ -126,3 +132,120 @@ func BagStateParDo() *beam.Pipeline {
 
 	return p
 }
+
+type combiningStateFn struct {
+	State0 state.Combining[int, int, int]
+	State1 state.Combining[int, int, int]
+	State2 state.Combining[string, string, int]
+	State3 state.Combining[string, string, int]
+	State4 state.Combining[int, int, int]
+}
+
+type accum1 struct{}
+
+func (ac *accum1) MergeAccumulators(a, b int) int {
+	return a + b
+}
+
+type accum2 struct{}
```

Comment:
accum* isn't a good name for these structs since they're combineFns and not accumulators.  comb* would be better as prefix.

Reply:
Yeah, that's fair - I updated to combine*